packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -151,6 +152,26 @@ function getCommonEventData<T extends keyof DocumentEventMap>(eventConfig: UIEve
     element = event.target instanceof HTMLElement ? event.target : null;
   }
 
+  let value: string | undefined;
+  const metadata: Metadata = {};
+  if (eventName === 'change' && element) {
+    switch (element.nodeName) {
+      case 'INPUT': {
+        const input = element as HTMLInputElement;
+        value = input.checked + '';
+        metadata.type = input.getAttribute('type') ?? '';
+        break;
+      }
+      case 'SELECT': {
+        const select = element as HTMLSelectElement;
+        value = select.value;
+        metadata.type = 'select';
+        metadata.text = select.options[select.selectedIndex].text;
```

Comment:
Does this need to be handled by the normal text logic?

Reply:
We don't really have any select element in the app to truly test it.. Usually the <option>s of <select> are predetermined, so it should be safe. 
We can revisit this later though.